cmd/fission-bundle/main.go

```diff
@@ -78,14 +78,11 @@ func runMessageQueueMgr(logger *zap.Logger, routerUrl string) {
 	}
 }
 
-func runStorageSvc(logger *zap.Logger, port int, filePath string) {
-	subdir := os.Getenv("SUBDIR")
-	if len(subdir) == 0 {
-		subdir = "fission-functions"
```

Comment:
Removing this might break backward compatibility.

Reply:
The idea is to remove all logic from `fission-bundle/main.go` to `storavesvc.go`  and make it consistent with other services in same file. Like 
```
func runMessageQueueMgr(logger *zap.Logger, routerUrl string) {
	err := mqtrigger.Start(logger, routerUrl)
	if err != nil {
		logger.Fatal("error starting message queue manager", zap.Error(err))
	}
}
```
Similarly storageSvc also has a `Start` function now which abstracts this logic. Since both statements are same ( earlier it was here and now moved to `storageSvc` so ideally this shouldn't break). Let me know if my understanding is correct?